openapi/integrator/token-requestor/bankaxept.yaml

```diff
@@ -63,6 +63,34 @@ paths:
         '503':
           $ref: '../components.yaml#/components/responses/503Error'
 
+  /v1/eligible-banks:
+    get:
+      summary: Eligible bank identifiers
```

Comment:
It is the banks that are eligible for enrolment with ePayment, not the identifiers, so I would prefer the summary `Eligible banks`.